src/cmap/auth/scram.ts

```diff
@@ -141,13 +135,8 @@ async function continueScramConversation(
   const username = cleanUsername(credentials.username);
   const password = credentials.password;
 
-  let processedPassword;
-  if (cryptoMethod === 'sha256') {
-    processedPassword =
-      'kModuleError' in saslprep || typeof saslprep !== 'function' ? password : saslprep(password);
-  } else {
-    processedPassword = passwordDigest(username, password);
-  }
+  const processedPassword =
+    cryptoMethod === 'sha256' ? saslprep(password) : passwordDigest(username, password);
```

Comment:
Unrelated to this line, The check deps github tests are failing, just need to update the expected list of deps.